src/aleph_client/asynchronous.py

```diff
@@ -392,13 +465,28 @@ async def forget(
         account: Account,
         hashes: List[str],
         reason: Optional[str],
-        storage_engine: StorageEnum = StorageEnum.storage,
-        channel: str = settings.DEFAULT_CHANNEL,
-        address: Optional[str] = settings.ADDRESS_TO_USE,
+        storage_engine: Optional[StorageEnum] = None,
```

Comment:
Why does `forget` specify a storage engine? Seems weird. I'm gonna give a go to refactoring the broadcast + submit code today, it's getting annoying.